test/lld/em_asm_O0.c

```diff
@@ -0,0 +1,7 @@
+#include <emscripten.h>
+
+int main() {
```

Comment:
You can avoid the `__original_main` thing by adding argc and argv here..  not important at all but would remove of of the differences.

Reply:
Thanks, that helps. I also sorted them differently which seems to help remove a little more of the diff.